specs/code-lists/CSV/country.csv

```diff
@@ -0,0 +1,250 @@
+Code,Label
```

Comment:
Was there a reason for choosing 'Label' over 'Title' (applies to the other codelists, too)? If we can use the latter that would be better as that is what the JSCC testing library expects.

Reply:
The reason was just that the existing codelists all used 'Label', I'll change the updated ones to 'Title' when I move them into their new folder